dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/relationship/hibernate/HibernateRelationshipStore.java

```diff
@@ -311,6 +311,19 @@ public List<Relationship> getByUids( List<String> uids )
         }
     }
 
+    @Override
+    protected void preProcessPredicates( CriteriaBuilder builder,
+        List<Function<Root<Relationship>, Predicate>> predicates )
+    {
+        predicates.add( root -> builder.equal( root.get( "deleted" ), false ) );
```

Comment:
does this mean all methods in here get this predicate added?

Reply:
All methods using HibernateGenericStore or HibernateIdentifiableObjectStore methods to retrieve objects from DB